apps/zbugs/api/push.ts

```diff
@@ -38,9 +40,42 @@ class Transaction implements DBTransaction<postgres.TransactionSql> {
 
 const mutatorSql = postgres(process.env.ZERO_UPSTREAM_DB as string);
 
+const transporter = nodemailer.createTransport({
+  host: 'smtp.ethereal.email',
+  port: 587,
+  auth: {
+    user: 'micaela35@ethereal.email',
+    pass: 'cq8t7nRzYW61YjmX56',
+  },
+});
```

Comment:
this'll be removed. Wanted to check mutators w/ code that can only run on the server.